src/transport/SecureSessionMgr.cpp

```diff
@@ -92,136 +92,157 @@ CHIP_ERROR SecureSessionMgr::SendMessage(NodeId peerNodeId, System::PacketBuffer
 CHIP_ERROR SecureSessionMgr::SendMessage(PayloadHeader & payloadHeader, NodeId peerNodeId, System::PacketBufferHandle msgBuf,
                                          EncryptedPacketBufferHandle * bufferRetainSlot)
 {
-    return SendMessage(payloadHeader, peerNodeId, std::move(msgBuf), bufferRetainSlot, false);
+    PacketHeader packetHeader;
+    return SendMessage(payloadHeader, packetHeader, peerNodeId, std::move(msgBuf), bufferRetainSlot, false);
 }
 
 CHIP_ERROR SecureSessionMgr::SendMessage(EncryptedPacketBufferHandle msgBuf, EncryptedPacketBufferHandle * bufferRetainSlot)
 {
+    CHIP_ERROR err = CHIP_NO_ERROR;
```

Comment:
consider using ReturnMacros.h instead of the gotoexit macros, which would allow you to declare variables where used instead of requiring C-style declaraction at the top.